### PR TITLE
Add errors parameter for CsvItemExporter with tests

### PR DESCRIPTION
**Pull request by maranqz**
Additional context from https://github.com/scrapy/scrapy/issues/4754

### CODE REVIEW

**Review by Gallaecio — COMMENTED** [review]
Looks good to me, only the documentation update is missing: https://docs.scrapy.org/en/latest/topics/exporters.html#csvitemexporter

File: docs/topics/exporters.rst [hunk 1]
```diff
@@ -311,6 +311,10 @@ CsvItemExporter
       multi-valued fields, if found.
    :type include_headers_line: str
 
+    :param errors: The optional string that specifies how encoding and decoding errors are to be handled.
+        For more information see `documentation <https://docs.python.org/3/library/io.html#io.TextIOWrapper>`_.
```
Comment (Gallaecio) [inline docs/topics/exporters.rst:315]: Please try building the documentation locally with `tox -e docs` to verify, but I think using InterSphinx here should work:

```suggestion
        For more information see :class:`io.TextIOWrapper`.
```

**Review by Gallaecio — APPROVED** [review]

**Comment by kmike** [comment]
Hey! Just to make sure - is my understanding is correct?

* As implemented, this is a feature which is not easily accessable for the end users
* to make it available through FEEDS option https://github.com/scrapy/scrapy/issues/4606 needs to be fixed, which was out of scope for this PR

**Comment by maranqz** [comment]
@kmike Yes, you are right. I think to do that task https://github.com/scrapy/scrapy/issues/4606 in the month.

**Comment by kmike** [comment]
Thanks @maranqz!

**Pull request merged**